File: Fortran/0075/0075_0053.f90
module m1
  contains
   subroutine foo(a)
    integer,contiguous::a(:)
    if (any(a/=[1,3])) print *,101,a
    a=[11,12]
   end subroutine
subroutine ss2(b)
integer,pointer:: b(:)
call foo(b(:))
end subroutine
end
subroutine s1
use m1
integer,target:: c(3)=[1,2,3]
integer,pointer:: p(:)
p=> c(::2)
call ss2(p     )
if (any(c/=[11,2,12])) print *,102,c
end
call s1
print *,'OK'
end 

File: C/0075/0075_0053.c
#include <stdio.h>
int foo(int j,int k)
{
     int i[10];
     i[j]=1;
     if ( j ) j=j+i[k];
     else     j++;
     return j;
}
int main() { printf("%d\n",foo(1,1)) ; }
